py_interp/src/py_interp_fortran.F90

```diff
@@ -333,6 +333,95 @@ SUBROUTINE compute_mslp(data_out, pres_field, psfc, ter, tk, qv, ix, iy, iz, it)
 END SUBROUTINE compute_mslp
 
 !---------------------------------------------------------------------
+SUBROUTINE calcslptwo(slp, PP, P_s, PHI_s, T_L, nz, ns, ew, nt)
+! Josipa's subroutine
```

Comment:
Where did this code really come from? Which formula is being implemented? please, add refererence
@Yoselita

Reply:
The code follows the algorithm described in chapter 4 of "FULL-POS in the cycle 38 of ARPEGE/IFS" by Yessad K. (METEO-FRANCE/CNRM/GMAP/ALGO), 2011. The original equation, that can be found in the section 4.3.1 "Mean sea level pressure Πmsl (routine PPPMER)" has been used in a Fortran 95 postprocessing script, implemented and under development by K. Goergen at al. The script converts standard WRF output to the CORDEX experiment protocol compliant data set ("CMORization"), and was adapted and used in several institutions that were running WRF for various CORDEX experiments.